includes/admin/class-wcmp-export-pdf.php

```diff
@@ -10,61 +10,26 @@
 
 class WCMP_Export_Pdf
```

Comment:
Is dit bestandje wel waardevol? Zou het allemaal duidelijker worden als dit helemaal weg kan?

Reply:
✅